src/components/Nav/index.js

```diff
@@ -124,7 +124,6 @@ class Nav extends React.Component {
         >
           <ul>
             <li><Link onClick={() => this.closeMenu()} to='/' title='Startseite'>Startseite</Link></li>
-            <li><Link onClick={() => this.closeMenu()} to='/about' title='Über uns'>Über uns</Link></li>
```

Comment:
Should it not suffice to just remove this menu entry? I do not care if there is a deep link that someone can find by accident.